src/donejs-carousel-plugin.js

```diff
@@ -538,6 +536,8 @@ export const ViewModel = DefineMap.extend({
 		}
 		this.listenTo(window, 'resize', () => {
 			this.isDesktop = isDesktop(); // set view on resize event
```

Comment:
@jvercide-tc Not going to block the PR over this, but at some point we should really throttle these `window.onresize` events :S

Reply:
@cjardine-tc yeah, we should implement the window.service you made on ting-node here. probably better as a separate ticket so this is not blocked :)